catalog/core/catalog-core-camelcomponent/src/main/java/ddf/camel/component/catalog/content/ContentProducer.java

```diff
@@ -121,13 +151,25 @@ public void process(Exchange exchange)
 
         if (StringUtils.isNotEmpty(mimeType)) {
             ContentItem newItem;
-            newItem = new ContentItemImpl(Files.asByteSource(ingestedFile), mimeType,
-                    ingestedFile.getName(), null);
+            newItem = new ContentItemImpl(Files.asByteSource(ingestedFile),
+                    mimeType,
+                    ingestedFile.getName(),
+                    null);
 
             LOGGER.debug("Creating content item.");
 
-            CreateStorageRequest createRequest = new CreateStorageRequestImpl(
-                    Collections.singletonList(newItem), null);
+            Map<String, Serializable> propertiesList = new HashMap<>();
+
+            headerMap.entrySet()
+                    .forEach((entry) -> {
+                        if (!CAMEL_DEFAULT_HEADERS.contains(entry.getKey())) {
```

Comment:
Since the `CreateStorageRequest` is brand new here, there doesn't seem to be any risk of passing malicious parameters down to the framework.  I'd like to get more opinions on this, if possible.